lib/EntrypointScriptBuilder.py

```diff
@@ -505,7 +506,9 @@ def _get_normalized_chart_repo_url(self):
         return normalized_repo_url
 
     def handle_non_plugin_repos(self):
-        helm_push_command = 'curl -u $HELMREPO_USERNAME:$HELMREPO_PASSWORD -T $PACKAGE ' + self.chart_repo_url + '$(basename $PACKAGE)'
+        if not self.chart_subdir.endswith('/'): #adding trailing slash to CHART_SUBDIR
+            self.chart_subdir += '/'
+        helm_push_command = 'curl -u $HELMREPO_USERNAME:$HELMREPO_PASSWORD -T $PACKAGE ' + self.chart_repo_url + self.chart_subdir + '$(basename $PACKAGE)'
```

Comment:
When CHART_SUBDIR will be empty it will fail the script because self.chart_subdir is None type and you can't add string to NoneType.